### PR TITLE
doc: Adding instructions for webhook integration with InfluxDB Cloud 2.0

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding the instructions for HTTP integration with InfluxDB Cloud 2.0 via Webhooks and Telegraf agent. 
Addition to #2968 

#### Testing
Tested with TTS and Telegraf running locally.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by adriansmares** [comment]
I'd suggest making a webhook template for this. Of course, the user still has to enter the base URL, but that should be the only variable for him.

**Comment by nejraselimovic** [comment]
@adriansmares I'm already working on it

File: doc/content/integrations/influxdb-cloud/telegraf-setup/telegraf-http-setup.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export INFLUX_TOKEN="paste your token here"
+```
+
+Run the Telegraf agent with this configuration file.
+
+In {{% tts %}} Console, [create a new webhook]({{< ref "/integrations/webhooks/creating-webhooks" >}}) with JSON **Webhook format**, set the **Base URL** to `http://localhost:8080/telegraf` and tick the box next to the message types you want to enable this webhook for.
+
+{{< figure src="../tts-webhook-info.png" alt="Creating webhook on The Things Stack" >}}
+
+>Note: keep in mind that Telegraf agent can be hosted somewhere other than your local machine. In that case, you need to adjust the **Base URL** according to your setup.
```
Comment (adriansmares) [inline doc/content/integrations/influxdb-cloud/telegraf-setup/telegraf-http-setup.md]: `can be hosted in a remote environment as well`.

**Review by adriansmares — APPROVED** [review]
LGTM with small comment. @laurensslats please review as well.

**Review by laurensslats — APPROVED** [review]
See Adrian's comment

**Pull request merged**